kview/src/main/resources/dev/ikm/komet/kview/controls/search-control.css

```diff
@@ -109,4 +110,101 @@
     -fx-text-fill: white;
     -fx-font-family: "Noto Sans";
     -fx-font-size: 14;
-}
\ No newline at end of file
+}
+
+.search-control > .search-results-list-view {
+    -fx-background-color: transparent;
+    -fx-padding: 0;
+}
+
+.search-control > .search-results-list-view > .placeholder > .placeholder-label {
+    -fx-font-family: "Noto Sans";
+    -fx-font-size: 12px;
+    -fx-text-fill: #FBFBFC;
+    -fx-font-weight: 500;
+}
+
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar {
+    -fx-background-color: transparent;
+    -fx-padding: 0;
+}
+
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar:vertical > .track {
+    -fx-background-color: transparent;
+    -fx-border-width: 0;
+    -fx-background-radius: 0em;
+    -fx-border-radius: 0;
+    -fx-background-insets: 0;
+}
+
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar:vertical > .thumb {
+    -fx-background-color: transparent, #F2F2F299;
+    -fx-background-insets: 0, 1;
+    -fx-background-radius: 2, 2;
+}
+
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar:vertical > .increment-button,
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar:vertical > .decrement-button,
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar:vertical > .increment-button > .increment-arrow,
+.search-control > .search-results-list-view > .virtual-flow > .scroll-bar:vertical > .decrement-button > .decrement-arrow {
+    -fx-background-color: transparent;
+    -fx-background-radius: 0em;
+    -fx-padding: 0 1 0 1;
+}
+
+.list-cell {
+    -fx-padding: 2.5 2 2.5 0;
+}
+.list-cell,
+.list-cell:odd,
+.list-cell:filled:selected,
```

Comment:
Can we make these styles more specific? By more specific I mean adding the parents, etc so that there's little probably of this style clashing with other styles defined in Komet.

Reply:
done

Reply:
Thanks!